framework/Samplers/Sampler.py

```diff
@@ -298,7 +298,7 @@ def readSamplerInit(self,xmlNode):
         self.initSeed = Distributions.randomIntegers(0,2**31,self)
         for childChild in child:
           if childChild.tag == "limit":
-            self.limit = childChild.text
+            self.limit = int(childChild.text)
```

Comment:
Possibly this should be in a try catch ValueError block (like Samplers.MonteCarlo does) to give a better error.